src/server/API_documentation.js

```diff
@@ -19,6 +19,19 @@ returns:
   "guest": false
 }
 
+-------------------------
+Check users email & password (LOGIN)
+-------------------------
+endpoint: '/users/login'
+request type: get
+additional body params: {
+  email: string,
+  password: string,
+}
+returns: true || false
+// for "security" sake, sending back sensitive info to the client side felt unsafe
+// so checking for email and password match in the back-end and returning back boolean
+
```

Comment:
Thanks Eric! I think it is better to send the password to the back end to resolve. Could it also return the userId so that I can store that into state?

Reply:
done!